index.html

```diff
@@ -217,6 +217,37 @@ <h6>LET'S CONNECT</h6>
             </li>
         </ul>
     </section>
+    <footer class="contact-form">
+        <h2>contact me</h2>
+        <div class="par">
+          <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        </div>
+        <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- Hey, please, let's fix the issue with the form URL. you should get a valid id from the [formspree service](https://formspree.io/html) to make the form submit correctly.